models/userSchemas.ts

```diff
@@ -0,0 +1,44 @@
+const mongoose = require('mongoose');
+
+enum userRole {
```

Comment:
Should we be exporting these enums? This way it is more intuitive to ask for `user role = admin` or `user-role = writer` rather than comparing the user role to integers